components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-service-provider.jsp

```diff
@@ -1451,12 +1486,16 @@
                                                                        type="password" autocomplete="off"
                                                                        id="oauthConsumerSecret"
                                                                        name="oauthConsumerSecret"
+                                                                       <% if (!(isHashDisabled != null && !isHashDisabled.isEmpty() && isHashDisabled.equals("false")) || appBean.getOauthConsumerSecret() == null) { %>
                                                                        value="<%=Encode.forHtmlAttribute(oauthConsumerSecret)%>"
+                                                                       <% } %>
                                                                        readonly="readonly">
+                                                                <% if (!((isHashDisabled != null && !isHashDisabled.isEmpty() && isHashDisabled.equals("false")) || appBean.getOauthConsumerSecret() == null)) { %>
                                                                 <span style="float: right;">
                                 						<a style="margin-top: 5px;" class="showHideBtn"
                                                            onclick="showHidePassword(this, 'oauthConsumerSecret')">Show</a>
```

Comment:
Shall we add i18n here?

Reply:
We can't add i18n here. bcz showHidePassword function is written to get the text value of the element as .text().
If we add i18n here , then the text value will be <fmt:message key='show'/>.
For the first time if we click on show buttton, it will not work.